squad/frontend/views.py

```diff
@@ -176,6 +164,49 @@ def project_badge(request, group_slug, project_slug):
     return HttpResponse(badge, content_type="image/svg+xml")
 
 
+def __badge_text(request):
```

Comment:
Did you mean "__badge_style()"?

Reply:
yes, we can call it that. I'll change it.